tests/e2e/e2e_test.go

```diff
@@ -317,4 +328,62 @@ var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
 			color.Outf("{{green}}add-node failed as expected")
 		})
 	})
+
+	ginkgo.It("can start with custom config", func() {
+		ginkgo.By("stopping network first", func() {
```

Comment:
Do we need to shut down the existing network for this? Can't we just start a new, separate network?

Reply:
We can't. Starting a new network results in an error "already bootstrapped" - at least that was my experience.

Reply:
To give you a bit more context @gyuho -- what we'd like to do is have an E2E test that makes sure that the `--custom-node-configs` flag works. To do that, we need to start a new network with that flag. What's the best way to do that (start a new network) in the E2E tests?